internal/rbd/types/volume.go

```diff
@@ -20,8 +20,10 @@ import (
 	"context"
 
 	"github.com/container-storage-interface/spec/lib/go/csi"
+	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
+//nolint:interfacebloat // more than 10 methods are needed for the interface
```

Comment:
In #4719  I added the `journalledObject` interface. Such an approach also prevents the need for skipping the `interfacebloat` linter.